Patch is applied because autotools are not used by upstream author. Patch setting prefix=/usr and fix install commands.

Index: zita-convolver/libs/Makefile
===================================================================
--- zita-convolver.orig/libs/Makefile	2009-12-15 23:30:27.000000000 +0100
+++ zita-convolver/libs/Makefile	2009-12-15 23:34:13.000000000 +0100
@@ -17,9 +17,9 @@
 
 # Modify as required.
 #
-PREFIX = /usr/local
+PREFIX = /usr
 SUFFIX := $(shell uname -m | sed -e 's/^unknown/$//' -e 's/^i.86/$//' -e 's/^x86_64/$/64/')
-LIBDIR = lib$(SUFFIX)
+LIBDIR = lib
 
 
 MAJVERS = 2
@@ -29,8 +29,8 @@
 
 
 CPPFLAGS += -Wall -I. -fPIC -D_REENTRANT -D_POSIX_PTHREAD_SEMANTICS -O3
-CPPFLAGS += -mmmx -msse -mfpmath=sse -ffast-math -funroll-loops -fpermissive
-CPPFLAGS += -march=i686
+CPPFLAGS += -ffast-math -funroll-loops
+#CPPFLAGS += -march=i686
 #CPPFLAGS += -DVECTORIZE 
 
 LDLFAGS += 
@@ -50,12 +50,12 @@
 
 
 install:	$(ZITA-CONVOLVER_MIN)
-	install -d $(PREFIX)/include
-	install -d $(PREFIX)/$(LIBDIR)
-	install -m 644 $(ZITA-CONVOLVER_H) $(PREFIX)/include
-	install -m 755 $(ZITA-CONVOLVER_MIN) $(PREFIX)/$(LIBDIR)
-	ldconfig
-	ln -sf $(ZITA-CONVOLVER_MIN) $(PREFIX)/$(LIBDIR)/$(ZITA-CONVOLVER_SO)
+	install -d $(DESTDIR)$(PREFIX)/include
+	install -d $(DESTDIR)$(PREFIX)/$(LIBDIR)
+	install -m 644 $(ZITA-CONVOLVER_H) $(DESTDIR)$(PREFIX)/include
+	install -m 755 $(ZITA-CONVOLVER_MIN) $(DESTDIR)$(PREFIX)/$(LIBDIR)
+	/sbin/ldconfig -n $(DESTDIR)$(PREFIX)/$(LIBDIR)
+	ln -sf $(ZITA-CONVOLVER_MIN) $(DESTDIR)$(PREFIX)/$(LIBDIR)/$(ZITA-CONVOLVER_SO)
 
 
 clean:
